kustomize/apps/training-operator/base/deployment.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       containers:
         - command:
             - /manager
-            - --mpi-kubectl-delivery-image=k8scc01covidacr.azurecr.io/mpioperator/kubectl-delivery:latest
+            - --mpi-kubectl-delivery-image=k8scc01covidacrdev.azurecr.io/mpioperator/kubectl-delivery:latest
```

Comment:
not sure if this should be using dev acr

Reply:
Updated